opentracing-cdi-example/src/main/java/io/opentracing/contrib/cdi/example/AccountService.java

```diff
@@ -0,0 +1,20 @@
+package io.opentracing.contrib.cdi.example;
+
+import io.opentracing.contrib.cdi.Traced;
+
+import java.util.logging.Logger;
+
+/**
+ * This is a simple synchronous EJB, without any knowledge about span context or other OpenTracing semantics. All it
+ * does is specify an interceptor and it's shown as the child of a parent span.
+ *
+ * @author Juraci Paixão Kröhling
```

Comment:
Maybe encode the non-ascii unicode characters?
I have been bitten many times with source files edited by different people using different IDE's on different operating systems.

Opinions on `@author` vary; @adriancole has asked me in previous reviews to drop the tag altogether.